frontend/app/src/App.test.tsx

```diff
@@ -1135,265 +1135,6 @@ describe("App", () => {
-        window.history.pushState({}, "", "/?foo=bar")
-
-        sendForwardMessage("newSession", NEW_SESSION_JSON)
-
-        expect(window.history.pushState).toHaveBeenLastCalledWith(
-          {},
-          "",
-          "/?foo=bar"
-        )
-      })
-
-      it("retains embed query params even if the page hash is different", () => {
-        const embedParams =
-          "embed=true&embed_options=disable_scrolling&embed_options=show_colored_line"
-        window.history.pushState({}, "", `/?${embedParams}`)
-
-        renderApp(getProps())
-
-        const hostCommunicationMgr = getStoredValue<HostCommunicationManager>(
-          HostCommunicationManager
-        )
-
-        const appPages = [
-          {
-            pageScriptHash: "toppage_hash",
-            pageName: "streamlit app",
-            urlPathname: "streamlit_app",
-          },
-          {
-            pageScriptHash: "subpage_hash",
-            pageName: "page2",
-            urlPathname: "page2",
-          },
-        ]
-
-        // Because the page URL is already "/" pointing to the main page, no new history is pushed.
-        sendForwardMessage("newSession", {
-          ...NEW_SESSION_JSON,
-          appPages,
-          pageScriptHash: "toppage_hash",
-        })
-
-        const navLinks = screen.queryAllByTestId("stSidebarNavLink")
-        expect(navLinks).toHaveLength(2)
-
-        // TODO: Utilize user-event instead of fireEvent
-        // eslint-disable-next-line testing-library/prefer-user-event
-        fireEvent.click(navLinks[1])
-
-        const connectionManager = getMockConnectionManager()
-
-        expect(
-          // @ts-expect-error
-          connectionManager.sendMessage.mock.calls[0][0].rerunScript
-            .queryString
-        ).toBe(embedParams)
-
-        expect(hostCommunicationMgr.sendMessageToHost).toHaveBeenCalledWith({
-          type: "SET_QUERY_PARAM",
-          queryParams: embedParams,
-        })
-      })
-
-      it("works with baseUrlPaths", () => {
-        renderApp(getProps())
-        vi.spyOn(
-          getMockConnectionManager(),
-          "getBaseUriParts"
-        ).mockReturnValue({
-          pathname: "/foo",
-          hostname: "",
-          port: "8501",
-        } as URL)
-
-        sendForwardMessage("newSession", {
-          ...NEW_SESSION_JSON,
-          appPages: [
-            {
-              pageScriptHash: "page_script_hash",
-              pageName: "streamlit app",
-              urlPathname: "streamlit_app",
-            },
-            {
-              pageScriptHash: "hash2",
-              pageName: "page2",
-              urlPathname: "page2",
-            },
-          ],
-          pageScriptHash: "hash2",
-        })
-
-        expect(window.history.pushState).toHaveBeenLastCalledWith(
-          {},
-          "",
-          "/foo/page2"
-        )
-      })
-
-      it("doesn't push a new history when the same page URL is already set", () => {
```

Comment:
Do we not need these tests anymore because it cannot happen in MPAv2? Just trying to collect some more context in case someone has to look into this again in the future

Reply:
Ah. I was originally going on a removing spree, but I stopped and decided to skip these in favor of "converting to MPA v2 tests" Will reintroduce and mark as skip and address this in the clean up PR.